var/spack/repos/builtin/packages/xyce/package.py

```diff
@@ -59,17 +61,30 @@ class Xyce(CMakePackage):
     variant("cxxstd", default="11", values=cxxstd_choices, multi=False)
 
     variant("pymi", default=False, description="Enable Python Model Interpreter for Xyce")
+    # Downstream dynamic library symbols from pip installed numpy and other
+    # pip installed python packages can cause conflicts. This is most often
+    # seen with blas symbols from numpy, and building blas static resolves
+    # this issue.
+    variant(
+        "pymi_static_blas",
+        default=True,
+        sticky=True,
+        description="Require static blas build for PyMi",
```

Comment:
I'd really like to keep this `pymi_static_blas` optional in the sense that I want users to be able to install with `+pymi` and not necessarily require that they use the static tpl constraints, which is why I default to `True` but am trying to avoid an explicit dependency between them.

Reply:
No, I mean this variant is meaningless when pymi isn't used right? So it should have when=

Reply:
Yes, you're right. As is, there is no logic where `+pymi_static_blas` does something without `+pymi`. I made the change. Thanks for the suggestion.